<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacher_main</class>
 <widget class="QWidget" name="teacher_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="teacher_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>511</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="info_frame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_user_name">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_user_id">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>用户id</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="btn_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>100</width>
      <height>360</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="info_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>个人信息</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="class_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>班级</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="setting_btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>设置</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="min_button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>21</width>
      <height>8</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>551</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="teacher_info_page">
     <widget class="QFrame" name="detail_frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>531</width>
        <height>361</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>511</width>
         <height>341</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>姓名：小明他王叔叔</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="3">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>一些图片</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>年龄：18</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>性别：沃尔玛购物袋</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>电话号码：110</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="3">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>备注：你无敌了孩子</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>对战积分：1000</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>学校：北京邮电大学包子铺</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="class_page">
     <widget class="QScrollArea" name="scrollArea_class">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>461</width>
        <height>351</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="class_contents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>459</width>
         <height>349</height>
        </rect>
       </property>
       <widget class="QLabel" name="class_label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>141</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>暂未创建班级</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="class_add_btn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="class_detail_page">
     <property name="toolTipDuration">
      <number>-7</number>
     </property>
     <widget class="QFrame" name="class_detail_contents">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>361</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_teacher_name">
       <property name="geometry">
        <rect>
         <x>29</x>
         <y>60</y>
         <width>171</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>teacher name</string>
       </property>
      </widget>
      <widget class="QScrollArea" name="student_scroll">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>481</width>
         <height>231</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="student_display">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>479</width>
          <height>229</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_class_name">
       <property name="geometry">
        <rect>
         <x>29</x>
         <y>20</y>
         <width>171</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>class name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_student_number">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string>学生人数:
42人</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QFrame" name="div_line">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>500</width>
         <height>2</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_del_class">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>40</y>
         <width>90</width>
         <height>40</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>个人信息</string>
       </property>
       <property name="text">
        <string>销毁班级</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_view_task">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>40</y>
         <width>90</width>
         <height>40</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>个人信息</string>
       </property>
       <property name="text">
        <string>管理任务</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="class_task_page">
     <widget class="QScrollArea" name="scrollArea_task">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>471</width>
        <height>301</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="task_contents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>469</width>
         <height>299</height>
        </rect>
       </property>
       <widget class="QLabel" name="task_label">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>140</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>暂无任务</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_class_of_task">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>班级： xxx的任务列表</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="btn_frame_4">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>70</y>
        <width>51</width>
        <height>121</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_add_task">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_return_to_detail">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <zorder>btn_frame_4</zorder>
     <zorder>scrollArea_task</zorder>
     <zorder>label_class_of_task</zorder>
    </widget>
    <widget class="QWidget" name="class_create_task">
     <widget class="QScrollArea" name="word_scroll">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>471</width>
        <height>301</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="word_contents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>469</width>
         <height>299</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_create_task">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>任务创建</string>
      </property>
     </widget>
     <widget class="QFrame" name="btn_frame_5">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>70</y>
        <width>51</width>
        <height>171</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_add_word">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_del_word">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_create_to_task">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_commit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>确认提交</string>
      </property>
     </widget>
     <widget class="AcrylicFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>请输入任务名</string>
       </property>
      </widget>
     </widget>
     <zorder>frame</zorder>
     <zorder>btn_frame_5</zorder>
     <zorder>word_scroll</zorder>
     <zorder>label_create_task</zorder>
     <zorder>btn_commit</zorder>
    </widget>
    <widget class="QWidget" name="task_condition_page">
     <widget class="QLabel" name="label_finish_count">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>当前已完成人数: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_task_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>task name:</string>
      </property>
     </widget>
     <widget class="QFrame" name="btn_frame_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>70</y>
        <width>51</width>
        <height>271</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_del_task">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_condition_to_content">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_lower">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_greater">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_condition_to_task">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView_student">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>470</width>
        <height>300</height>
       </rect>
      </property>
     </widget>
     <zorder>btn_frame_2</zorder>
     <zorder>label_finish_count</zorder>
     <zorder>label_task_name</zorder>
     <zorder>tableView_student</zorder>
    </widget>
    <widget class="QWidget" name="task_content_page">
     <widget class="QScrollArea" name="scrollArea_task_view">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>471</width>
        <height>301</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="task_view_contents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>469</width>
         <height>299</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="btn_frame_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>70</y>
        <width>51</width>
        <height>151</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_del_task_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_content_to_task">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_content_to_condition">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_task_content">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>任务单词总览</string>
      </property>
     </widget>
     <zorder>btn_frame_3</zorder>
     <zorder>scrollArea_task_view</zorder>
     <zorder>label_task_content</zorder>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>171</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_tips">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>141</width>
        <height>351</height>
       </rect>
      </property>
      <property name="text">
       <string>提示:文本文件要求每个单词使用空格或换行符分开</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_loadtxt">
         <property name="text">
          <string>加载文本文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_addtotask">
         <property name="text">
          <string>添加到任务</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_discard">
         <property name="text">
          <string>放弃添加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ret">
         <property name="text">
          <string>返回上级</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_teacher_main_pic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>教师端</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AcrylicFrame</class>
   <extends>QFrame</extends>
   <header location="global">acrylicframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
